antiplag/tasks.py

```diff
@@ -41,6 +46,18 @@ def process_documents(submission_id):
     submission.status = Submission.SubmissionStatus.PROCESSED
     submission.save()
 
+    # send email when done
+    if submission.email is not None:
+        try:
+            validate_email(submission.email)
+        except ValidationError as e:
+            print(_("bad email, details:"), e)
+        else:
+            send_mail(_("Antiplag - Your check has finished!"),
+                _("Check the results of your check at https://antiplag.sk/result/") + str(submission.id) + "/" ,
```

Comment:
What about using string interpolation instead of concatenation?

Reply:
Not sure if this is supported with the translation service, do you know?

Reply:
Yes it's possible, look at the file size / file number check.